### PR TITLE
KardinoDX.TouchHLEG version 2.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100051)

### CODE REVIEW

**Comment by Trenly** [comment]
image

**Review by Trenly — CHANGES_REQUESTED** [review]
This may not be a good fit for WinGet due to the requirement of touchHLE and touchHLEG being extracted to the same folder; You may need to set `InstallLocationRequired: true` for this package
image

**Comment by stephengillie** [comment]
> image

I'm not sure how it passed validation, as the application still shows this message when launched.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

File: manifests/k/KardinoDX/TouchHLEG/2.0/KardinoDX.TouchHLEG.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: KardinoDX.TouchHLEG
+PackageVersion: 2.0
```
Comment (stephengillie) [inline manifests/k/KardinoDX/TouchHLEG/2.0/KardinoDX.TouchHLEG.installer.yaml:5]:
```suggestion
PackageVersion: 2.0
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

**Pull request closed without merging**